src/extension.ts

```diff
@@ -1048,37 +1049,40 @@ async function selectPod(scope: PodSelectionScope, fallback: PodSelectionFallbac
     return value.pod;
 }
 
-async function logsKubernetes(explorerNode?: explorer.ResourceNode) {
+async function logsKubernetes(explorerNode?: explorer.ResourceNode, follow?: boolean) {
```

Comment:
I'm a bit allergic to Booleans in this context, because the meaning is not obvious at point of call.  (One of my colleagues wrote http://www.nichesoftware.co.nz/2016/05/15/code-gardening-bools-are-evil.html which pretty much sums up my feelings.)  I'd suggest we declare an enum and make it mandatory.

Reply:
Thanks for the reference article, really helps to understand the refactor path.